dep/nuget/packages.config

```diff
@@ -8,7 +8,8 @@
   <package id="Microsoft.VCRTForwarders.140" version="1.0.4" targetFramework="native" />
   <package id="Microsoft.Internal.Windows.Terminal.ThemeHelpers" version="0.6.220404001" targetFramework="native" />
   <package id="Microsoft.VisualStudio.Setup.Configuration.Native" version="2.3.2262" targetFramework="native" developmentDependency="true" />
-  <package id="Microsoft.UI.Xaml" version="2.7.3" targetFramework="native" />
+  <package id="Microsoft.UI.Xaml" version="2.8.2" targetFramework="native" />
+  <package id="Microsoft.Web.WebView2" version="1.0.1661.34" targetFramework="native" />

```

Comment:
TODO Dustin: look at the content of this package and see if we can suppress the DLL being copied in.

Reply:
Looked at, fixed.